### ISSUE TITLE
Unable to save notes

### ISSUE THREAD

**Issue by joeg1484** [nuxsmin/sysPass#196]
Not sure when this started as I have not had to create a note for a while, and there have been several updates during this time, but I am unable to save notes on accounts I have created or am a member of and edits are enabled.

I even tired git pull to get the latest and it still doesnt work.

What I have done is created a new account and tried to add a note.. The note went into the text box, but when I clicked the save button, and when back in, there was no note there. I can see where the account was changed in the revert box, but no notes.

I also tried this is an already existing account where I have notes previously. I try to add a note, and can type in the text box, but the changes wont save.

Let me know if you need any more information.

I am using CentOS 6.5 as the server and Firefox latest on the client. I have also tried Chrome.

Thanks!
Joe

**Comment by nuxsmin** [comment]
Hi Joe, can you give me some info about your environment?

That is very strange because all the tests that I've done are fine in the latest version (try out at demo.syspass.org)

Kind regards

**Comment by joeg1484** [comment]
Ni Nuxsmin,

Sure, what do you want to know...

I have the following installed on the server:
PHP
`
php-pecl-apc-3.1.15-0.6.svn329913.el6.art.x86_64
php-pear-Mail-Mime-1.8.4-1.el6.noarch
php-imap-5.4.45-53.el6.art.x86_64
php-xml-5.4.45-53.el6.art.x86_64
php-pear-1.9.4-4.el6.noarch
php-mbstring-5.4.45-53.el6.art.x86_64
php-pear-Net-SMTP-1.6.1-1.el6.noarch
php-cli-5.4.45-53.el6.art.x86_64
php-mysql-5.4.45-53.el6.art.x86_64
php-mcrypt-5.4.45-53.el6.art.x86_64
php-ldap-5.4.45-53.el6.art.x86_64
php-pear-Net-IDNA2-0.1.1-3.el6.noarch
php-common-5.4.45-53.el6.art.x86_64
php-pdo-5.4.45-53.el6.art.x86_64
php-5.4.45-53.el6.art.x86_64
php-xmlrpc-5.4.45-53.el6.art.x86_64
php-soap-5.4.45-53.el6.art.x86_64
php-pear-Net-Socket-1.0.10-1.el6.noarch
php-devel-5.4.45-53.el6.art.x86_64
php-snmp-5.4.45-53.el6.art.x86_64
php-pear-Mail-mimeDecode-1.5.5-1.el6.noarch
php-odbc-5.4.45-53.el6.art.x86_64
php-gd-5.4.45-53.el6.art.x86_64
php-tidy-5.4.45-53.el6.art.x86_64
php-pear-Auth-SASL-1.0.6-1.el6.noarch
`

For Apache:
`httpd-2.2.15-47.el6.centos.1.x86_64`

MySQL is 
`
mysql-server-5.5.47-32.el6.art.x86_64
php-mysql-5.4.45-53.el6.art.x86_64
`

Its a CentOS release 6.7 (Final) system

My client is numerous systems with various clients from IE to Firefox, Safari to Chrome.

I looked at my log files and there was nothing really interesting there:
These were present then I was using tail -f and accessing...

`192.168.0.1 - - [02/Feb/2016:20:47:04 -0700] "POST /sysPass/ajax/ajax_getContent.php HTTP/1.1" 200 1291 "https://myhost.com/sysPass/index.php" "Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:43.0) Gecko/20100101 Firefox/43.0"`

I can do a screen cast to show you if you want me too. Let me know what other info you need.

Thanks for the help!
Joe

**Comment by nuxsmin** [comment]
Thanks, it seems that you're running it in a tested environment some time ago...

What about the sysPass version?

**Comment by joeg1484** [comment]
Hi There,

I am using the git version:

commit 3fa8e5a8721dfd411df8761591cae5c42210a4e9
Author: nuxsmin <nuxsmin@syspass.org>
Date:   Mon Jan 11 16:30:36 2016 +0100

Thanks!
Joe

**Comment by nuxsmin** [comment]
Can you take a look to the Apache error.log when you performs the request? (saving an account)

What about to dig into the database and get if the notes are stored in the accounts table?

**Comment by st3reo** [comment]
Hi
I can also confirm this bug. Notes are not saving.
I'm running a fresh install of sysPass 1.2.0.11 on a CentOS 7 with php 5.6 and apache 2.4
I can provide more details if needed.
Thanks

**Comment by nuxsmin** [comment]
Hi there @joeg1484 and @st3reo 

sorry about the response delay...I'm so busy in the last time :(

About this annoying issue, is very strange because I've tested it on several environments and it works fine (try out at the demo website), indeed, I'm using it on a production environment.

Can you take a look to the database accounts table?. You can issue this command in the mysql cli:

<code>
SELECT account_notes FROM accounts WHERE account_name = 'THE_ACCOUNT_NAME';
</code>

**Comment by st3reo** [comment]
@nuxsmin thanks a lot for your reply.
Looking a bit into this it seems that the notes dissapearing bug is only manifesting while using the "Classic" theme, and works fine with the "Material Blue" one. I have also searched in the DB using the command provided by you and indeed there is no note stored in the DB.
I have also tested this on your demo site ( demo.syspass.org ) and it seems to be the same thing there too.
